site/mkdocs.yml

```diff
@@ -72,7 +72,9 @@ nav:
     - Maintenance Procedures: spark-procedures.md
     - Structured Streaming: spark-structured-streaming.md
     - Time Travel: spark-queries/#time-travel
-  - Trino: https://trino.io/docs/current/connector/iceberg.html
+  - Presto:
+    - Trino (PrestoSQL): https://trino.io/docs/current/connector/iceberg.html
+    - PrestoDB: https://prestodb.io/docs/current/connector/iceberg.html
```

Comment:
Hm, I'm wondering how we want to handle this. There should be more and more engines with Iceberg support and we don't want to keep adding tabs for each one. Maybe there's room for PrestoDB? Or maybe we should fix the problem now and have a page for documentation that is hosted elsewhere.

Reply:
Yes, I think we need to find a better way in organizing tabs. I am putting up a new PR, we can discuss there.